cartridges/app_custom_cartridge/cartridge/scripts/productList/productListExport.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+
+'use strict';
+
+/**
+ * To find the products having the productInfo attribute and
+ * sorting it inside a file through a job
+ * @param {string} jobParameter job parameters
+ */
+'use strict';
+
+/**
+* To find the products having the productInfo attribute and
+* sorting it inside a file through a job
+* @param {string} jobParameter job parameters
+*/
+function process(jobParameter) {
+    var ProductMgr = require('dw/catalog/ProductMgr');
+    var FileWriter = require('dw/io/FileWriter');
+    var File = require('dw/io/File');
+
+    var fileName = jobParameter.exportFile || 'default_product3';
```

Comment:
give some appropriate name like 'products_export'

Reply:
changed the fileName as mentioned